places-solution-integrations/README.md

```diff
@@ -0,0 +1,23 @@
+# Places Solution Integrations
+
+By itself, using Adobe Places to collect location data isn't very useful in your app. You will start to get real value from location data when you provide it to your other Adobe solutions. This will allow you to use the location data to deliver custom experiences and make informed decisions.
+
+This section will cover how to integrate Adobe Places with other AEP SDK extensions so you can put your location data to work.
```

Comment:
Please reword to:
"This section provides information about how to integrate Places with other Mobile SDK extensions, so you can put your location data to work."